source/sass/buyers-guide/views/category.scss

```diff
@@ -7,7 +7,7 @@
     margin: 0 auto;
     max-width: 545px;
 
-    @media screen and (min-width: 750px) {
+    @media screen and (min-width: $bp-md) {
```

Comment:
Not directly related to this code change but we actually don't need the class `page-width`. We can just move all the CSS rules `.category-listing`.

Reply:
That... is a good point.